VoteCounter/src/VoteCount.cs

```diff
@@ -74,7 +104,24 @@ private void findVotes(Post post)
 
         private bool isVoteValid(string vote)
```

Comment:
Maybe break this into:
bool isVoteeValid(out votee)
bool isVoterValid [Ticket #11]